clouddriver-openstack/src/test/groovy/com/netflix/spinnaker/clouddriver/openstack/client/OpenstackOrchestrationV1ClientProviderSpec.groovy

```diff
@@ -71,10 +90,10 @@ class OpenstackOrchestrationV1ClientProviderSpec extends OpenstackClientProvider
 
     then:
     1 * stackApi.create(_ as StackCreate) >> { StackCreate sc ->
-      sc.disableRollback == stackCreate.disableRollback
-      sc.name == stackCreate.name
-      sc.parameters == stackCreate.parameters
-      sc.template == stackCreate.parameters
+      assert sc.disableRollback == stackCreate.disableRollback
```

Comment:
Assert is needed inside of closures. If you leave it off, the assertions do not validate anything.